surl/__init__.py

```diff
@@ -395,7 +387,7 @@ def get_client(web_login, store_env, store_type):
         if store_type == "snapcraft"
         else CONSTANTS[store_env]["pubgw_base_url"],
         storage_base_url="https://storage.staging.snapcraftcontent.com",
-        user_agent=DEFAULT_HEADERS["User-Agent"],
+        user_agent="surl/devel",
```

Comment:
Why not keep the old `user_agent="surl/{}".format(os.environ.get("SNAP_VERSION", "devel"))` ?

Reply:
Purely due to a lack of braining things :)